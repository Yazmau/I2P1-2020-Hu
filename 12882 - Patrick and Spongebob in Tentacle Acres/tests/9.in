87 67
13 4 56 51 100 56 43 85 66 20 13 70 29 1 54 57 16 97 10 42 18 43 24 59 57 50 81 53 31 86 1 52 5 80 71 43 3 73 80 33 69 10 96 31 29 19 88 83 90 87 77 10 3 62 100 92 46 88 16 83 61 71 79 95 97 20 49
4 69 34 72 38 63 26 10 53 16 30 14 35 79 46 11 68 11 58 23 57 34 54 98 25 50 40 25 67 83 34 71 27 3 33 91 65 49 67 75 44 54 82 35 35 36 15 12 100 30 24 94 64 73 14 43 25 90 31 29 60 57 33 20 87 96 37
5 68 94 90 56 36 92 15 67 25 55 92 1 44 28 74 88 9 47 71 75 82 1 15 41 75 83 25 48 84 36 91 30 45 54 41 8 33 73 82 22 13 42 47 80 70 34 46 37 62 57 53 26 60 80 16 68 15 27 19 35 31 31 29 83 15 89
10 51 68 25 14 49 69 29 27 19 45 23 4 97 77 42 16 45 19 98 35 74 80 55 54 90 62 42 97 75 71 44 72 85 56 5 59 71 19 50 85 6 50 91 22 37 9 38 9 39 28 67 11 35 18 46 66 15 35 60 46 14 7 78 31 55 82
100 34 16 2 55 60 23 7 51 14 12 42 14 67 28 18 79 40 12 22 24 35 56 6 73 97 37 53 15 28 68 52 38 22 68 66 30 71 88 50 94 35 98 29 96 6 9 53 95 22 4 82 26 77 77 89 41 61 99 60 59 34 20 96 23 3 23
19 10 41 76 66 96 45 63 72 54 63 93 33 34 10 71 28 78 85 92 78 97 85 20 33 19 29 19 89 71 69 54 82 12 15 83 48 77 57 80 81 60 100 20 27 76 19 21 52 90 86 3 60 20 48 56 90 65 33 62 95 87 12 60 38 74 79
29 98 18 14 16 4 48 27 34 14 5 69 39 4 50 72 93 71 74 51 51 93 27 39 28 27 54 9 2 99 42 55 59 2 69 8 79 23 20 40 44 42 26 80 32 54 100 48 35 47 44 72 2 65 21 38 37 83 80 59 26 54 27 50 100 63 68
39 93 4 43 85 60 59 89 1 96 21 90 93 95 97 64 90 36 47 21 6 45 97 90 54 94 93 90 13 8 97 31 40 15 94 5 92 16 34 13 79 16 50 67 83 15 36 56 93 13 33 13 11 37 49 66 67 29 66 33 90 76 73 32 45 44 61
70 24 12 66 19 50 54 3 95 94 66 16 95 77 22 81 56 30 32 5 14 20 39 68 6 88 6 6 29 92 40 14 79 93 26 15 57 42 32 22 35 46 56 46 78 71 34 14 10 85 90 27 46 67 97 92 4 43 52 48 81 41 91 59 71 87 92
69 13 28 81 14 21 80 74 44 39 10 13 37 4 38 68 66 96 56 94 80 52 51 38 7 56 22 74 14 10 99 69 37 85 75 50 99 67 75 35 20 24 90 43 40 9 97 80 50 78 29 2 86 24 5 78 4 63 97 76 39 52 92 73 18 5 39
53 86 29 23 3 54 100 11 80 33 41 84 12 4 95 44 5 30 22 92 14 30 29 25 53 68 54 6 73 28 51 76 60 42 25 41 18 56 56 70 49 27 16 77 35 3 7 60 8 46 81 96 5 41 94 62 85 71 95 6 92 80 5 79 91 51 45
84 5 96 34 30 81 79 59 7 26 43 70 29 89 12 98 89 35 17 90 5 41 23 71 30 66 8 90 47 52 85 48 64 63 89 56 31 68 27 9 71 87 20 25 12 98 95 27 36 64 99 42 57 88 96 7 4 39 94 95 40 53 81 34 81 76 83
66 66 87 57 14 83 4 59 35 91 71 59 46 32 9 74 96 26 82 99 65 11 47 57 62 66 25 5 52 76 84 18 58 89 24 92 90 25 96 74 84 96 22 8 58 89 89 30 82 36 79 98 75 64 11 50 77 30 28 95 12 71 92 35 42 50 41
65 34 64 12 31 85 66 17 27 4 31 80 55 8 96 36 31 73 66 86 87 51 67 2 83 4 84 83 5 85 72 18 38 16 20 82 10 29 6 99 51 14 87 43 22 52 16 84 70 25 41 28 62 29 77 20 67 12 67 41 72 51 76 1 25 2 86
24 93 36 79 12 28 70 64 84 69 72 86 91 41 25 45 22 98 9 84 5 45 94 46 78 13 56 92 100 81 52 93 57 54 50 97 97 46 43 82 49 11 39 90 45 18 68 30 79 64 13 29 29 32 60 47 51 99 77 22 38 51 8 85 50 40 55
32 5 10 66 93 58 5 17 42 35 60 82 97 99 22 100 83 64 48 61 77 10 54 73 26 87 21 40 94 77 99 4 16 24 11 17 94 81 68 59 88 3 11 27 37 70 50 11 90 31 18 81 76 17 62 20 26 84 13 83 89 84 32 88 26 38 5
59 84 100 21 84 83 24 40 92 10 9 3 56 86 49 21 21 93 84 75 94 52 15 94 54 41 66 34 39 1 96 17 80 59 98 55 60 83 55 65 36 84 34 74 69 92 70 93 55 54 45 18 28 35 90 5 71 30 4 89 14 53 28 5 88 35 63
3 79 21 34 80 19 4 31 38 53 78 15 50 1 77 10 10 17 7 53 36 92 32 45 61 51 3 53 99 60 33 67 75 78 80 75 52 66 69 76 50 44 35 42 65 63 62 14 79 49 68 30 89 80 80 68 2 40 2 100 83 23 16 63 85 35 72
33 82 55 39 51 49 15 72 49 73 83 43 38 14 72 47 63 15 43 16 60 81 1 90 90 55 41 96 50 36 68 41 93 17 32 7 22 72 1 6 83 98 67 78 32 85 35 63 43 51 90 6 82 65 92 23 36 13 70 85 12 65 73 48 89 41 88
39 8 41 74 82 65 28 38 70 86 50 29 52 90 55 96 30 44 22 67 37 17 24 43 44 54 14 12 15 67 70 68 1 9 19 90 31 18 95 8 91 67 36 14 62 81 89 45 42 85 97 5 49 34 52 27 92 54 21 11 100 58 68 7 9 14 42
78 94 81 56 42 74 75 60 7 2 54 96 9 20 9 37 54 82 37 61 30 41 58 9 93 67 63 24 16 21 77 46 37 49 97 74 18 37 15 97 78 55 39 41 29 59 28 74 74 84 80 88 12 75 14 82 58 78 11 73 66 79 12 8 91 98 15
54 38 99 94 71 53 17 32 50 91 12 39 12 54 80 78 88 54 91 76 9 95 7 63 67 2 90 71 51 24 100 69 82 6 17 56 98 75 40 71 57 46 81 1 44 89 48 43 13 95 76 6 93 43 89 18 23 70 30 88 27 89 56 70 60 36 12
57 6 65 37 30 13 7 52 70 88 55 42 8 47 87 74 31 30 74 80 17 40 37 39 69 92 93 88 30 3 82 57 90 80 1 71 40 55 15 12 46 26 49 66 72 47 34 73 91 84 72 77 26 94 91 62 29 19 72 50 89 39 95 43 38 35 85
90 9 29 39 9 46 82 2 62 71 60 49 10 58 12 53 95 59 29 72 2 32 15 78 77 32 65 27 58 73 43 28 59 49 1 38 100 1 38 41 84 41 58 15 52 9 1 62 42 59 9 5 55 3 88 48 10 94 56 11 63 7 9 83 12 85 54
57 76 59 44 1 79 51 82 68 47 91 43 82 98 67 7 41 6 33 30 65 35 87 48 37 37 17 99 85 75 92 29 2 13 56 91 31 39 83 50 60 55 84 94 66 56 45 62 12 45 94 42 77 50 27 33 72 4 74 47 93 43 22 4 79 7 73
11 89 21 75 84 68 73 17 27 64 14 71 58 63 2 94 39 92 50 15 36 73 32 10 93 38 41 48 37 51 87 45 93 6 59 47 99 7 40 8 49 78 59 84 96 76 13 13 25 95 48 38 32 89 85 81 67 97 88 25 39 34 64 64 32 46 15
53 12 61 24 99 64 87 64 13 11 89 76 36 60 32 86 44 33 89 96 36 47 72 91 61 27 81 94 26 80 38 56 79 30 98 49 38 72 39 74 80 31 93 49 42 9 90 26 74 59 44 80 89 92 18 33 68 87 91 1 52 1 20 83 12 1 39
12 35 86 67 26 55 32 61 43 47 7 66 23 14 74 66 29 14 50 91 39 38 65 8 74 85 7 78 67 36 49 67 60 32 44 18 15 25 86 88 22 88 86 81 46 86 11 96 84 99 41 11 59 84 48 88 31 29 67 83 44 64 82 6 77 60 21
84 97 27 86 22 42 24 27 68 17 17 65 49 84 46 30 59 96 29 49 72 14 80 60 28 18 15 53 96 56 7 66 33 89 7 39 43 12 87 36 78 66 88 33 41 77 49 64 14 62 17 14 63 39 15 63 60 45 61 33 58 87 9 61 89 78 55
61 51 95 13 63 8 12 22 21 48 47 27 31 56 69 73 61 25 56 69 70 43 74 94 30 35 68 39 28 49 40 82 9 55 75 55 59 36 99 46 79 42 58 1 28 89 42 44 45 6 99 58 63 53 64 47 8 51 42 4 17 68 75 17 86 2 69
75 45 16 96 35 70 52 95 25 93 90 88 62 31 55 23 66 33 82 11 82 64 74 58 56 76 34 12 46 88 68 39 7 51 51 22 50 70 45 25 33 56 58 19 15 21 78 81 26 44 79 4 58 49 97 10 84 83 80 25 43 23 29 27 51 3 74
73 5 76 81 57 40 93 33 47 43 97 69 51 33 85 47 82 28 59 7 44 74 52 81 89 55 11 14 4 33 56 27 61 43 59 33 25 14 6 11 57 3 26 60 51 55 70 34 33 77 99 76 99 12 66 57 13 95 69 58 95 55 67 100 43 80 11
60 55 24 1 69 32 58 57 5 88 73 57 89 57 74 43 64 4 50 70 48 86 65 40 13 47 68 34 88 98 44 72 66 46 81 73 62 56 4 76 54 6 69 6 66 54 74 61 40 69 22 98 87 49 31 20 60 80 12 99 40 98 32 19 35 6 41
74 6 98 2 21 50 90 5 35 83 73 74 80 43 92 18 84 72 77 52 78 49 70 96 21 18 43 91 6 83 100 60 72 90 56 2 87 73 51 52 26 60 45 12 9 93 80 73 47 89 11 25 93 50 21 31 61 36 62 77 60 35 26 67 60 36 40
98 14 84 49 39 30 51 58 58 47 26 14 83 100 54 85 76 53 75 38 21 74 5 26 70 50 66 86 93 25 54 89 90 39 98 48 32 27 66 61 35 97 7 52 48 1 17 25 23 26 54 44 74 63 37 83 43 93 84 44 54 11 47 36 36 77 19
89 97 6 68 92 7 14 43 24 17 92 8 98 65 88 49 79 46 31 73 17 55 42 70 46 66 91 44 15 10 44 15 67 65 34 5 37 27 23 50 41 17 36 14 96 43 8 15 99 62 63 24 41 74 16 55 2 42 96 91 28 87 72 43 94 53 99
14 64 74 74 92 100 48 41 15 100 19 49 100 1 77 9 98 14 41 84 43 99 81 98 94 6 79 98 42 50 27 21 41 91 24 18 30 55 79 44 32 21 59 55 64 72 63 72 99 68 82 16 58 89 23 1 72 46 31 5 28 84 29 88 32 52 13
19 79 96 96 84 24 47 74 87 20 88 93 8 84 71 63 18 88 69 48 85 54 23 21 13 49 2 78 81 94 66 81 73 15 34 37 25 87 82 60 60 91 90 46 53 48 36 71 69 46 13 25 55 95 24 31 14 94 70 15 51 32 29 17 77 26 26
42 71 1 82 98 34 29 24 30 8 31 98 79 61 43 43 52 98 74 88 41 29 21 96 74 99 69 2 68 18 1 50 17 99 59 52 87 61 95 100 85 72 21 1 78 71 83 55 15 22 46 72 1 17 20 6 97 51 25 7 91 24 55 72 20 60 79
79 55 17 3 94 91 1 35 89 16 22 3 43 34 24 92 70 77 38 75 86 31 13 4 46 52 72 100 7 34 46 87 24 58 7 39 56 45 15 56 83 1 66 14 37 51 22 46 29 32 64 92 75 85 24 78 14 93 84 76 28 79 17 63 1 45 74
86 80 74 87 92 94 17 67 56 100 12 50 7 45 30 52 93 98 14 24 88 20 57 18 8 25 55 82 9 4 87 82 93 37 79 40 23 70 33 94 89 52 4 27 63 58 20 13 25 1 81 66 29 7 88 5 63 48 11 40 87 44 92 56 62 23 18
94 22 67 72 89 89 34 28 34 86 9 36 55 65 82 37 44 25 35 91 30 33 97 46 75 92 33 64 3 83 10 61 41 8 78 22 42 5 69 30 2 5 49 4 78 7 56 71 21 45 96 73 39 26 22 84 79 36 48 36 19 60 43 51 62 41 41
19 69 100 45 44 51 39 90 4 83 17 87 5 66 39 82 2 70 19 73 66 53 12 84 35 68 79 29 40 56 41 91 60 4 38 46 25 57 40 36 26 93 34 96 85 4 33 42 74 7 15 40 74 79 95 77 49 86 94 52 89 95 61 77 48 5 16
28 41 92 72 80 84 44 38 74 58 72 27 3 66 32 78 58 50 27 25 79 2 98 76 45 76 19 31 21 63 63 65 44 49 37 72 97 93 52 46 29 1 49 67 69 72 87 16 4 86 5 50 66 19 95 13 66 7 21 18 45 46 27 57 77 80 35
27 3 77 34 53 37 12 99 87 5 90 75 62 52 4 7 19 3 59 21 81 9 22 24 71 95 39 95 16 23 86 26 57 75 100 21 67 51 54 47 50 74 40 92 99 80 42 31 90 93 78 45 93 6 39 60 44 39 20 1 26 95 95 15 22 47 55
45 92 2 51 68 20 71 68 38 35 23 55 56 89 5 44 30 36 91 64 66 27 71 36 20 92 48 52 76 11 57 5 26 85 46 71 10 54 35 28 10 34 38 35 35 40 17 46 92 54 10 48 56 82 65 60 12 29 98 80 3 71 8 50 57 100 43
31 49 46 83 63 39 61 55 2 94 9 99 79 19 24 82 31 8 17 48 58 26 17 17 55 20 10 53 99 73 92 10 78 50 10 29 81 37 55 25 37 59 94 66 47 75 8 39 30 77 10 5 36 33 81 45 85 93 25 10 11 78 90 81 28 51 49
8 39 93 16 32 11 81 18 94 50 61 16 97 40 76 3 70 58 93 56 75 83 24 46 21 6 64 3 30 93 10 80 16 12 25 60 52 72 57 89 88 57 88 23 9 69 27 87 93 66 66 74 14 67 72 95 22 12 8 40 25 58 43 74 30 80 11
24 46 93 77 36 79 39 72 34 6 29 56 47 33 62 5 51 86 92 97 7 43 33 97 84 28 98 68 87 84 59 26 75 77 12 55 78 3 4 50 88 56 56 58 81 99 9 20 15 76 23 59 19 70 26 11 28 29 53 31 7 24 54 15 81 21 3
6 71 76 44 45 45 8 83 94 35 29 27 21 83 79 25 99 21 86 6 78 26 47 23 35 22 42 8 44 70 54 66 18 19 26 84 33 93 8 41 78 20 89 64 48 94 75 11 76 51 27 33 39 4 43 25 32 81 12 97 63 19 13 87 84 6 90
15 38 24 98 14 40 25 78 17 31 57 48 20 22 56 29 36 33 5 71 54 67 27 12 29 28 3 33 20 11 45 35 89 60 15 1 39 60 96 1 9 8 50 15 97 98 33 61 42 60 85 17 70 3 36 49 84 21 14 22 32 27 3 66 14 24 65
14 11 36 79 20 12 26 75 42 60 38 73 10 79 16 9 20 81 92 81 63 31 100 52 32 79 48 93 1 47 37 10 20 10 54 33 61 50 73 39 99 3 32 3 68 59 38 80 80 48 2 8 14 73 12 22 12 69 75 13 74 95 31 38 79 64 28
83 7 60 58 74 14 33 4 50 63 38 5 90 94 94 97 59 7 87 41 51 17 23 47 29 28 50 27 90 55 34 65 7 7 65 4 17 77 11 52 63 16 93 45 99 9 79 86 26 53 99 87 31 79 56 52 84 15 4 36 1 96 22 66 79 51 5
79 14 82 31 34 44 86 75 50 52 37 95 52 71 24 23 15 6 34 96 26 98 76 86 2 15 67 6 49 50 61 77 95 71 36 23 73 64 81 12 89 8 50 3 39 72 85 50 92 69 27 40 42 100 74 71 13 61 91 94 21 48 100 19 21 72 75
98 41 91 21 25 33 63 65 38 4 80 63 44 58 59 85 82 3 41 81 3 1 57 44 12 2 50 8 77 52 39 24 68 10 93 7 77 10 61 41 68 30 3 70 70 97 64 12 68 53 13 57 58 51 89 66 18 4 71 26 34 44 26 37 50 61 71
74 52 12 3 14 54 13 37 22 67 30 16 88 10 4 8 10 97 2 9 66 27 76 31 13 75 21 40 40 67 33 56 38 26 88 35 100 34 77 87 75 15 80 76 98 46 29 74 57 83 82 51 71 59 89 18 37 2 100 29 25 67 18 8 32 3 5
51 64 14 31 40 14 42 75 63 13 6 14 68 17 88 52 53 45 83 19 9 2 1 19 26 82 50 17 67 38 92 71 39 36 53 9 44 38 61 80 1 45 65 2 85 39 68 95 58 57 9 34 59 77 88 8 17 9 17 73 22 50 88 42 13 42 96
49 48 27 22 65 10 90 11 14 70 49 12 67 66 17 64 19 16 53 97 87 92 93 95 30 40 3 71 51 3 94 37 39 75 26 87 68 47 29 44 15 13 81 63 20 89 19 17 45 4 46 26 74 55 10 20 92 85 70 74 21 48 62 18 29 93 65
44 1 15 1 61 46 100 30 13 66 64 63 77 96 36 92 66 51 60 29 48 67 41 5 3 12 40 96 58 93 82 66 47 74 38 18 21 54 26 16 94 34 30 3 21 24 39 100 53 89 65 5 80 12 17 54 56 56 96 63 78 42 58 45 30 97 64
16 20 12 23 84 15 27 39 13 68 46 69 9 95 96 57 54 98 18 4 13 33 6 75 40 20 100 30 52 56 93 62 44 63 22 52 21 83 17 27 4 22 93 50 75 69 73 60 4 73 85 46 60 68 41 37 41 62 36 13 89 49 42 47 77 60 74
23 79 6 65 83 52 64 73 19 78 46 11 59 19 54 18 92 37 75 77 70 100 39 21 75 75 86 88 7 43 21 75 67 66 76 5 54 65 61 34 83 67 98 14 100 4 70 59 36 55 49 58 3 84 8 59 5 82 18 56 18 14 61 54 13 55 34
71 44 28 93 93 58 41 77 46 69 22 30 22 36 66 78 85 31 65 71 26 77 79 49 13 77 31 29 13 94 56 14 35 34 32 16 61 44 92 20 80 82 37 65 18 7 54 3 45 2 76 30 86 9 28 71 19 23 73 8 86 87 15 63 91 42 93
38 86 42 48 98 3 97 58 44 60 38 68 14 66 32 100 40 84 49 35 65 14 72 56 82 92 55 36 22 49 34 20 60 65 62 22 3 35 28 68 97 51 99 7 33 32 98 37 42 91 79 76 61 11 72 4 39 63 54 97 13 76 2 51 49 20 79
57 16 48 38 87 82 91 4 35 28 32 54 15 42 48 97 1 58 29 48 30 31 72 9 72 70 96 68 24 78 61 98 85 31 86 83 31 42 84 80 76 9 57 86 71 2 5 94 52 53 52 91 65 47 98 67 84 96 18 82 39 17 43 4 14 26 2
60 4 6 50 16 91 82 59 62 36 54 72 50 68 39 29 25 53 58 57 64 21 96 32 11 58 32 99 18 97 84 3 39 55 44 98 87 31 26 70 100 34 64 76 49 57 45 6 55 67 12 97 60 12 100 68 59 47 42 35 79 17 14 86 94 71 6
59 40 48 9 7 34 5 82 65 88 27 6 65 83 17 74 59 24 72 50 38 18 37 47 61 64 94 78 17 17 96 67 37 82 79 20 41 59 6 54 28 14 51 55 73 31 45 70 30 90 97 96 36 77 52 52 52 95 72 6 98 22 95 15 13 78 78
37 72 8 19 63 23 95 16 90 67 90 82 54 78 48 99 50 72 87 92 7 57 54 68 98 16 67 11 59 85 34 87 26 28 71 22 2 100 64 94 9 30 89 81 17 23 46 15 18 81 68 82 42 26 92 40 88 43 27 84 78 83 57 21 18 90 78
31 45 35 52 10 87 70 86 28 8 46 42 43 22 79 9 93 17 65 6 59 82 76 82 18 68 27 23 45 94 93 32 76 59 66 39 31 80 19 87 21 62 10 2 57 49 63 24 61 14 82 30 26 79 38 41 14 45 91 32 29 85 93 93 47 84 75
29 57 93 81 35 35 99 67 17 69 76 27 11 39 19 35 14 31 63 61 6 46 42 5 25 7 35 78 82 35 28 13 14 40 69 45 23 71 23 25 75 73 85 93 65 9 7 98 93 92 78 5 55 60 34 8 65 30 44 69 18 53 12 28 75 36 32
26 37 27 18 41 13 7 18 81 39 97 46 69 52 52 81 35 88 96 41 72 40 35 52 41 29 95 55 57 19 90 46 32 55 29 29 1 14 46 50 25 36 22 9 15 37 80 31 97 63 96 67 93 82 85 64 86 15 7 25 3 50 25 99 63 20 48
35 40 62 14 74 43 3 57 11 61 74 29 96 66 62 90 13 76 88 7 79 36 73 12 66 82 9 89 4 56 73 57 88 87 53 52 48 74 23 53 68 25 18 100 24 35 32 29 73 84 9 40 49 28 69 38 67 30 26 63 12 62 92 58 25 28 80
71 34 38 21 6 8 2 41 76 100 41 53 69 38 81 53 25 83 65 62 66 99 8 3 73 88 45 87 29 68 61 61 19 71 4 75 40 91 80 36 34 39 33 70 75 53 96 92 80 40 59 21 23 42 74 12 85 6 26 73 9 75 76 14 72 80 54
64 95 6 67 2 88 73 70 53 17 31 47 3 16 68 46 95 85 60 50 63 31 100 7 3 15 93 41 35 91 87 70 16 68 28 48 59 12 79 1 88 65 13 60 53 37 87 79 81 4 42 56 86 17 77 14 3 81 59 86 10 62 50 20 47 58 81
70 59 65 31 8 19 11 14 11 74 59 29 57 6 89 23 46 72 89 85 18 3 5 16 73 9 47 84 86 78 57 94 45 55 8 11 89 68 1 67 14 11 83 15 88 15 47 75 30 85 30 61 32 26 42 44 73 99 30 22 18 32 25 50 97 70 19
22 31 5 25 30 11 18 56 44 8 50 30 2 63 25 68 4 31 37 9 19 91 96 25 26 87 89 67 81 11 60 60 5 54 57 18 29 5 97 42 68 46 97 66 49 37 41 60 6 62 90 20 41 69 65 14 86 17 58 38 51 6 13 78 87 95 32
44 59 40 43 5 68 85 55 80 55 49 12 24 27 100 43 61 83 49 89 32 65 85 41 50 25 82 58 94 43 97 15 4 81 12 49 86 11 17 64 94 28 70 14 10 56 50 63 55 8 86 64 23 72 65 12 48 1 87 10 94 76 16 97 56 13 54
21 32 37 42 65 74 32 68 68 34 27 2 51 59 72 5 66 52 61 23 46 90 93 30 33 4 95 56 38 46 53 77 77 91 6 53 89 8 40 23 49 94 30 23 37 51 52 48 71 50 82 99 28 30 89 58 55 83 22 66 82 94 88 77 76 80 74
9 64 62 7 59 45 24 29 69 43 33 26 81 44 9 58 39 97 56 3 31 65 8 92 24 25 78 42 31 27 93 10 38 2 46 57 69 76 3 54 100 92 48 72 91 80 69 6 45 21 78 89 97 46 73 29 17 68 92 76 14 53 64 81 16 16 99
27 28 90 55 51 33 89 77 53 14 92 67 92 32 88 22 22 21 62 80 33 46 70 43 57 22 33 32 99 56 37 14 67 92 68 40 51 27 48 83 86 55 98 33 86 52 70 39 10 82 61 80 29 28 59 67 33 55 10 78 37 69 81 61 7 62 93
79 38 94 67 53 82 21 80 61 27 10 76 45 66 56 60 58 60 56 24 73 28 21 13 28 81 27 32 78 82 85 89 27 21 19 29 30 65 32 21 27 32 92 85 28 76 12 93 84 6 57 32 52 63 61 12 40 29 18 91 48 9 37 77 88 71 24
19 27 79 28 5 37 91 64 61 89 91 76 51 56 27 64 31 12 61 18 46 42 21 7 20 99 2 46 83 95 84 47 63 82 96 46 97 17 74 50 80 79 64 10 31 17 44 59 81 53 26 44 75 84 27 97 45 96 39 49 41 79 32 31 47 21 17
89 28 66 28 92 31 18 4 53 3 59 24 52 26 7 70 40 71 28 69 100 90 30 3 20 39 28 48 85 42 91 69 65 74 12 54 60 17 39 24 78 49 27 63 46 87 63 6 80 31 65 46 100 49 97 15 77 59 90 19 11 1 69 76 90 11 40
78 21 18 60 83 51 91 48 2 29 27 44 20 30 18 99 18 66 24 25 33 96 27 45 21 87 34 57 31 25 91 49 22 87 83 15 85 34 37 46 39 34 58 38 71 9 8 66 66 17 74 95 91 31 14 30 98 81 63 5 84 49 59 83 89 3 50
40 19 34 61 13 24 100 25 30 40 42 9 79 24 41 3 50 92 92 24 12 94 91 31 48 69 14 50 6 31 67 9 93 76 92 73 2 8 51 63 27 4 84 26 33 86 26 64 2 77 42 5 47 14 87 15 7 73 44 33 75 93 89 64 36 12 26
20 32 84 79 36 49 64 39 90 44 32 29 68 71 76 81 58 72 1 72 81 88 41 24 34 4 66 17 45 64 43 10 35 47 11 33 48 76 51 7 49 28 32 25 83 98 59 94 74 53 52 18 55 54 84 84 91 51 54 57 42 93 33 72 24 35 60
23 27 96 78 76 55 50 20 10 45 21 53 36 17 20 37 89 20 93 18 6 63 32 93 79 94 39 3 50 33 61 86 27 82 46 27 15 21 17 19 20 47 58 3 91 68 37 50 2 37 57 19 74 46 97 28 50 77 99 100 38 57 44 26 25 64 57
48 91 18 81 69 68 76 41 59 67 26 42 14 60 4 67 35 34 81 56 14 34 95 28 65 47 37 32 6 79 9 21 80 36 14 14 55 29 56 48 11 20 83 93 45 35 11 94 75 76 75 92 63 36 41 18 28 69 95 37 97 11 18 91 36 23 86
41
18 60 52 64
51 54 19 27
62 65 48 61
70 76 21 44
36 53 62 65
87 87 54 55
38 51 51 57
70 85 38 54
9 61 48 50
8 34 17 42
73 81 58 58
56 79 29 42
73 85 52 53
21 49 48 59
55 82 56 57
23 49 51 54
59 79 15 29
73 74 14 56
46 71 24 45
44 56 40 47
32 63 39 57
84 86 26 48
19 20 51 57
36 41 6 12
2 76 66 66
76 82 3 9
8 79 45 51
21 37 43 55
18 35 19 58
64 77 24 50
29 45 48 57
38 66 20 59
35 46 52 56
67 83 58 66
69 83 48 54
20 87 49 56
62 75 43 49
56 61 59 61
27 62 61 61
27 77 58 58
77 87 11 65
