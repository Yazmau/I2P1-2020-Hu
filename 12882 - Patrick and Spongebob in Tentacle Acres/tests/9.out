100
100
100
100
96
41
97
100
100
100
99
100
100
99
97
99
100
100
100
99
100
98
97
100
100
94
100
99
100
100
99
100
95
99
100
100
100
100
95
99
100
